### DISCUSSION TITLE
amamba/quickstart/scan-with-pipeline/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1886] [Announcements]
# amamba/quickstart/scan-with-pipeline/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/amamba/quickstart/scan-with-pipeline/

**Comment by muma378** [comment]
文档第5步的地址jenkins里配置SQ的地址的时候不能以 / 结尾，会导致QualityGate解析的SQ的URL多一个/报错
